homeassistant/components/climate/strings.json

```diff
@@ -123,6 +123,17 @@
         "swing_modes": {
           "name": "Swing modes"
         },
+        "swing_horizontal_mode": {
+          "name": "Swing horizontal mode",
```

Comment:
While I think we should leave service name etc. as is, this probably should all be "Horizontal swing ..." in translations as it sounds strange